.github/workflows/smoketest.yml

```diff
@@ -61,3 +61,23 @@ jobs:
         run: npm install
       - name: Lint
         run: npm run-script lint
+  build-test:
```

Comment:
I can't add inline comments for code outside the diff, but on line 14 it looks like this is configured for pull requests into `master`. I'm sure github aliases that with `main`

Reply:
Ooh, good catch! I'm pretty sure that they don't, actually. On the other hand, the pull request target is mostly there for show￼￼— if we just run the test on all pushes to all branches, it doesn't matter if you do or don't have a pull request.